openmdao/drivers/scipy_optimizer.py

```diff
@@ -569,6 +570,8 @@ def _objfunc(self, x_new):
 
             # Pass in new parameters
             i = 0
+            if MPI:
+                x_new = model.comm.bcast(x_new, root=0)
```

Comment:
Should probably replace bcast with Bcast.  bcast pickles the array before sending it, so it's slower.

Reply:
Changed